.github/workflows/python-publish-pypi.yml

```diff
@@ -28,3 +28,15 @@ jobs:
       run: |
         python setup.py sdist bdist_wheel
         twine upload dist/*
+
+  publish:
+    runs-on: ubuntu-latest
+
```

Comment:
Should we add a `needs` here to gate on a successful run of units so that we don't accidentally auto-deploy a broken version?

Reply:
This should be fixed now.